Description: Fix a parallel build FTBFS
 Files were sometimes written to trans/ before that directory was created.
 Based on similar patch for tuxpaint-config
Author: Adrian Bunk <bunk@debian.org>
        Jonathan Carter <jcc@debian.org>

--- tuxpaint-0.9.22.orig/Makefile
+++ tuxpaint-0.9.22/Makefile
@@ -410,7 +410,7 @@ endif
 
 # Build the translation files for gettext
 
-$(MOFILES): trans/%.mo: src/po/%.po  
+$(MOFILES): trans/%.mo: src/po/%.po trans
 	msgfmt -o $@ $<
 
 .PHONY: translations
